REQUIRE

```diff
@@ -7,3 +7,5 @@ SortingAlgorithms
 Reexport
 Compat 0.19.0
 FileIO 0.1.2
+CSV
+DataStreams
```

Comment:
DataStreams is required but it's also in the REQUIRE file for CSV. So this is a duplicate and unnecessary